aspnetcore/fundamentals/minimal-apis/responses.md

```diff
@@ -140,12 +140,20 @@ The following sections demonstrate the usage of the common result helpers.
 
 :::code language="csharp" source="~/fundamentals/minimal-apis/9.0-samples/Snippets/Program.cs" id="snippet_06":::
 
+:::moniker range=">= aspnetcore-9.0"
```

Comment:
I added the moniker range because the Internal Server Error is added in v9.
I also added a problem details example, which was also added in v9.

Is this OK? Or is it preferred to move the existing docs to a v8 page and copy the content to a v9 page?

Reference: https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?view=aspnetcore-9.0#minimal-apis